mne/viz/tests/test_3d.py

```diff
@@ -217,8 +220,17 @@ def test_plot_alignment(tmpdir, renderer, mixed_fwd_cov_evoked):
     renderer.backend._close_all()
     # no-head version
     renderer.backend._close_all()
+    # trans required
+    with pytest.raises(ValueError, match='transformation matrix is required'):
+        plot_alignment(info, trans=None, src=src_fname)
```

Comment:
This should not raise an error, as the default for `plot_alignment` is `coord_frame='head'` and it should be possible to plot all channel positions in the head coordinate frame without requiring a head<->MRI transformation

Reply:
The source space is in "mri"...

Reply:
Ahh yes I missed that!